packages/url_launcher/url_launcher_web/pubspec.yaml

```diff
@@ -26,3 +26,8 @@ dependencies:
 dev_dependencies:
   flutter_test:
     sdk: flutter
+
+topics:
+  - os-integration
+  - url-launcher
+  - urls
```

Comment:
Would it also make sense to add "link" or "links"? It's a more common term on the web compared to URL.

Reply:
Good idea! I'll add it to all of the url_launcher packages.

Reply:
(The fact that there's no consistency in topics between singular and plural makes me sad; hopefully that will shake out over time...)